infra/core/database/azure-cache-for-redis.bicep

```diff
@@ -105,11 +115,21 @@ param redisCacheCapacity int = 1
 @description('If set, the private endpoint settings for this resource')
 param privateEndpointSettings PrivateEndpointSettings?
 
+@description('Specify name of Built-In access policy to use as assignment.')
+@allowed([
+  'Data Owner'
+  'Data Contributor'
+  'Data Reader'
+])
+param builtInAccessPolicyName string = 'Data Reader'
+
+param users RedisUser[] = []
+
```

Comment:
Combine the accessPolicyName into the RedisUser object.

Reply:
i did that but hadn't pushed that change yet (still figuring out how to type things in bicep). I'll include it in this pr